src/lib/vip-import-validate-files.js

```diff
@@ -0,0 +1,366 @@
+// @flow
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+import fs from 'fs';
+import path from 'path';
+import { promisify } from 'util';
+
+// Promisify to use async/await
+const readDir = promisify( fs.readdir );
+
+// Accepted media file extensions
+export const acceptedExtensions = [
+	'jpg', 'jpeg', 'jpe',
+	'gif',
+	'png',
+	'bmp',
+	'tiff', 'tif',
+	'ico',
+	'asf',
+	'asx',
+	'wmv', 'wmx', 'wm',
+	'avi',
+	'divx',
+	'mov',
+	'qt',
+	'mpeg', 'mpg', 'mpe', 'mp4', 'm4v',
+	'ogv',
+	'webm',
+	'mkv',
+	'3gp', '3gpp', '3g2', '3gp2',
+	'txt',
+	'asc',
+	'c', 'cc', 'h',
+	'srt',
+	'csv', 'tsv',
+	'ics',
+	'rtx',
+	'css',
+	'vtt',
+	'dfxp',
+	'mp3',
+	'm4a', 'm4b',
+	'ra',
+	'ram',
+	'wav',
+	'ogg',
+	'oga',
+	'mid', 'midi',
+	'wma',
+	'wax',
+	'mka',
+	'rtf',
+	'js',
+	'pdf',
+	'class',
+	'psd',
+	'xcf',
+	'doc',
+	'pot',
+	'pps',
+	'ppt',
+	'wri',
+	'xla', 'xls', 'xlt', 'xlw',
+	'mdb', 'mpp',
+	'docx', 'docm', 'dotx', 'dotm',
+	'xlsx', 'xlsm', 'xlsb', 'xltx', 'xltm', 'xlam',
+	'pptx', 'pptm', 'ppsx', 'ppsm', 'potx', 'potm', 'ppam',
+	'sldx', 'sldm',
+	'onetoc', ' onetoc2', 'onetmp', 'onepkg', 'oxps',
+	'xps',
+	'odt', 'odp', 'ods', 'odg', 'odc', 'odb', 'odf',
+	'wp', 'wpd',
+	'key', 'numbers', 'pages',
+];
+
+/** Recommendations
```

Comment:
Not sure what is common styling for comments on the team just yet, but possibly consider moving this comment down one line so that it isn't on the same line as the opening comment syntax `/**`

Reply:
Yes, fixed in d74a527!